config.js

```diff
@@ -121,6 +121,7 @@ config.ENDPOINT_MONITOR_INTERVAL = 10 * 60 * 1000; // 10min
 // Keep connection on long requests
 config.S3_KEEP_ALIVE_WHITESPACE_INTERVAL = 15 * 1000;
 config.S3_MD_SIZE_LIMIT = 2 * 1024;
+config.SEMAPHORE_MONITOR_INTERVAL = 10 * 1000;
```

Comment:
As discussed, add a config for the period we want an average for (e.g. 5 min).
check if you can implement with an array of intervals, so we can get metrics for multiple averages (e.g., 1 min, 5 min, 15 min). In that case, test if `load_config_env_overrides` in this file works for a numbers array.

Reply:
@dannyzaken Once we get the average of multiple intervals how are we planning to push this value? In different metrics? same metrics with different labels?